.github/workflows/docs.yml

```
@@ -0,0 +1,34 @@
+name: Docs
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip 
+          pip install mkdocs-material
+
+      - name: Generate docs 
+        run: |
+          cp -rf README.md docs
+          mkdocs build --clean
```

We could use directly the mkdocs-material docker image and avoid installing python manually:
```suggestion
          docker run --rm -v ${PWD}:/docs squidfunk/mkdocs-material:6.2.4 -- build
```
WDYT?

---

Since the docker image already exists and we are not using any hype plugin, I think this would fit perfect